onnxruntime/test/python/transformers/test_parity_mixtral_moe.py

```diff
@@ -352,7 +397,7 @@ def test_mixtral_moe_parity(self):
         for batch_size in [1, 16]:
             for sequence_length in [128, 1024]:
                 # use a small sizes to speed up the test
-                config = MixtralConfig(hidden_size=256, intermediate_size=1024)
+                config = MixtralConfig()
```

Comment:
let's create two tests in TestMixtralMoE class similar to test_parity_moe.py. one with small inputs for parity purpose and one with large inputs for benchmark purpose but marked with @pytest.mark.slow so that the pipeline won't run it.